gammapy/cube/make.py

```diff
@@ -217,16 +233,22 @@ def __init__(
         self.exclusion_mask = exclusion_mask
         self.background_oversampling = background_oversampling
         self.maps = {}
+        self.migra_axis = (
+            migra_axis
+            if migra_axis
+            else MapAxis(nodes=np.linspace(0.0, 1.0, 11), unit="", name="migra")
```

Comment:
Are these default values for the `migra` and `rad` axis reasonable?

Reply:
Maybe declare the default `migra_axis` and `rad_axis` at the top of the file as `RAD_AXIS_DEFAULT` and `MIGRA_AXIS_DEFAULT` os they can be imported? I'm pretty sure we'll need these defaults in other places as well (e.g. `MapDataset.create()`)...

I'm not sure what good default are, I think we've never done areal study how good the edisp and psf should be sampled. As far as I know the default values for the FITS data export are:

```
RAD_MAX = 0.6656057834625244
RAD_AXIS_DEFAULT = MapAxis.from_bounds(0, RAD_MAX, nbin=144, node_type="edges", name="theta", unit="deg")
MIGRA_AXIS_DEFAULT = MapAxis.from_bounds(0.2, 5, nbin=160, node_type="edges", name="migra")
``` 
But this seems heavily oversampled...maybe @registerrier has an idea?

Reply:
I discussed a bit with @bkhelifi and have put some new defaults for now. This is relatively coarse, but can be a starting point.

Reply:
I would say that the default should the binning used in the IRF. It is safe, it can be too large, but a knowledgeable user could change it.